addon-test-support/index.js

```diff
@@ -25,8 +25,9 @@ export default function setupLifelineValidation(hooks) {
 
   hooks.afterEach(function(assert) {
     try {
-      let retainedObjects = [...registeredDisposables.keys()].map(o =>
-        o.toString()
+      let retainedObjects = [];
+      registeredDisposables.forEach((_, k) =>
+        retainedObjects.push(k.toString())
```

Comment:
Required because we don't use the polyfill anymore and IE11 does not support `Map#keys`, but `Map#forEach`.